### PR TITLE
Update assetlist.json

### PR DESCRIPTION
**Pull request by seisaicoin**
{
  "Name": "Seis AI",
  "description": "Top Secret AI reasoning technology memecoin, more than a meme",
  "symbol": "SEIS",
  "base": "0xE94a7c93A00B12E803166D8e355F617B920DD48e",
  "display": "SEIS",
  "denom_units": [
    {
      "denom": "0xE94a7c93A00B12E803166D8e355F617B920DD48e",
      "exponent": 0
    },
    {
      "denom": "SEIS",
      "exponent": 18
    }
  ],
  "images": {
    "png": "https://raw.githubusercontent.com/seisaicoin/SEIS/refs/heads/main/seis%20AI%20token%20logo.png"
  },
  "type_asset": "sdk.coin"
}

### CODE REVIEW

**Review by seisaicoin — COMMENTED** [review]
token logo update

File: assetlist.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         }
       ],
       "images": {
-        "png": "https://raw.githubusercontent.com/Sei-Public-Goods/sei-assetlist/main/images/Sei.png"
+        "png": "https://raw.githubusercontent.com/seisaicoin/SEIS/refs/heads/main/seis%20AI%20token%20logo.png"
```
Comment (nullmames) [inline assetlist.json:20]: You have changed the logo for the sei token. Please revert this change.

File: assetlist.json [hunk 2]
```diff
@@ -1035,3 +1035,24 @@
     }
   ]
 }
+{
```
Comment (nullmames) [inline assetlist.json:1038]: Please add this block of json to the appropriate network array.

Comment (seisaicoin) [reply assetlist.json:1038]: I apologize. Let me redo please

**Review by seisaicoin — COMMENTED** [review]
Appreciate and updated.

**Comment by nullmames** [comment]
> Appreciate and updated.

I do not see any changes committed to the PR

**Comment by seisaicoin** [comment]
Okay I did it again through the fork, am hoping it went through this time.

**Comment by nullmames** [comment]
Closing for https://github.com/Sei-Public-Goods/sei-assetlist/pull/22
